### ISSUE TITLE
Simple Payments Guided Tour uses blue text on black background

### ISSUE THREAD

**Issue by beaulebens** [Automattic/wp-calypso#22759]
On a new site/account (not sure which), you get a Guided Tour when you go to the Editor, telling you about the Simple Payment button. It uses blue text on black background which is really hard to read (especially on a lower quality screen). Suggest we make that link white.

![screen shot 2018-02-22 at 7 02 55 pm](https://user-images.githubusercontent.com/108942/36600685-2a7c74cc-1870-11e8-8b40-bc21caefe94c.png)

**Comment by designsimply** [comment]
I tried testing creating a new site from an existing site and also doing a new sign up (tested in production) and didn't get the Simple Payments Guided Tour ([2m13s](https://cloudup.com/cmuu1BNkke1)).

**Comment by rickybanister** [comment]
@lsinger have you spotted this before?

**Comment by lsinger** [comment]
Yes, but I don't recall whether it was a conscious decision to make the link blue or whether it was maybe a remnant from when the tour steps themselves were lighter. I cross-checked with @shaunandrews and he also doesn't remember. We both agree that making the links white would be a good idea.

**Comment by keoshi** [comment]
Using the color that was already mentioned in that stylesheet `#a8bece`:

![image](https://user-images.githubusercontent.com/390760/37046231-ba0b287a-215f-11e8-9857-71fdc99a38aa.png)

How does this look, @lsinger @shaunandrews ?

**Comment by lsinger** [comment]
@keoshi tbh I wouldn't mind a brighter color (eg white ¯\_(ツ)_/¯) for a higher contrast.

**Comment by keoshi** [comment]
@lsinger sounds good, will create that PR in a second.
